<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Application_robot</class>
 <widget class="QMainWindow" name="Application_robot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1494</width>
    <height>963</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Application_robot</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_connect">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>261</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Capteurs</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>261</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_temperature_text">
        <property name="text">
         <string>Temperature</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_temperature_valeur">
        <property name="text">
         <string>.....</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_gaz_text">
        <property name="text">
         <string>Gaz</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_gaz_valeur">
        <property name="text">
         <string>......</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>metr</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>85</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>85</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Autre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>....</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>460</y>
      <width>301</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Robot</string>
    </property>
    <widget class="QPushButton" name="pushButton_gauche">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Gauche</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_droit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Droit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_avancer">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Avancer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_reculer">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Reculer</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_connect">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>480</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_camera">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>300</y>
      <width>241</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Camera</string>
    </property>
    <widget class="QLabel" name="label_tourelle_haut">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string> haut</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>     bas</string>
     </property>
    </widget>
    <widget class="QDial" name="dial_tourelle_haut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="maximum">
      <number>174</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>90</number>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDial" name="dial_tourelle_bas">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>101</width>
       <height>81</height>
      </rect>
     </property>
     <property name="maximum">
      <number>60</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="pageStep">
      <number>20</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="notchTarget">
      <double>10.000000000000000</double>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="cam">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>501</width>
      <height>391</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gyro_x_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="acc_x_label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="acc_y_label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Gaz_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ultrason_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="gyro_y_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="gyro_z_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>220</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="acc_z_label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="Temperature_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <zorder>cam</zorder>
   <zorder>pushButton_connect</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>label_connect</zorder>
   <zorder>groupBox_camera</zorder>
   <zorder>gyro_x_label</zorder>
   <zorder>acc_x_label</zorder>
   <zorder>acc_y_label</zorder>
   <zorder>Gaz_label</zorder>
   <zorder>Ultrason_label</zorder>
   <zorder>gyro_y_label</zorder>
   <zorder>gyro_z_label</zorder>
   <zorder>acc_z_label</zorder>
   <zorder>Temperature_label</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
